src/apis/rollupscan.ts

```diff
@@ -1,21 +1,19 @@
 import { requireEnv } from "@/utils"
 
-const baseUrl = requireEnv("REACT_APP_API_BASE_URI")
```

Comment:
@zzq0826 @Thegaram we should probably completely remove this env var to avoid confusion